### ISSUE TITLE
ETQ Super admin je veux que les modifications apportées dans le formulaire CRM soient automatiquement intégrées au formulaire de demande de compte admin

### ISSUE THREAD

**Issue by Benjamin-Doberset** [betagouv/tps#2397]
Pour se créer un compte admin, la personne doit renseigner un formulaire dont les informations seront automatiquement intégrées à l'offre dans notre CRM. Ce formulaire était auparavant configuré via l'outil CRM, mais il semble que les deux soient désormais décorrélés.

**Comment by gregoirenovel** [comment]
Oui, et c'est très compliqué de les synchroniser. Donc ça couterait cher pour des modifications assez occasionnelles. Pour modifier le formulaire, aller voir un dev :)

**Comment by LeSim** [comment]
Je reouvre la demande, je travaille dessus, j'ai retiré le terme `automatique`
